### PR TITLE
Flexibility Hub attributes inside Scipion objects

### PR DESCRIPTION
**Pull request by DavidHerreros**
Flexibility Hub attributes have been added to standard Scipion objects.

Method `copyInfo` and subsets have also been modified to copy extended attributes,

### CODE REVIEW

File: pwem/objects/data.py [hunk 1]
```diff
@@ -759,6 +764,23 @@ def setMRCSamplingRate(self):
         """ Sets the sampling rate to the mrc file represented by this image"""
         from pwem.convert.headers import setMRCSamplingRate
         setMRCSamplingRate(self.getFileName(), self.getSamplingRate())
+    def getZFlex(self):
```
Comment (pconesa) [inline pwem/objects/data.py:767]: Are all these needed by particle and volume?

File: pwem/objects/data.py [hunk 2]
```diff
@@ -1295,6 +1318,14 @@ def hasCTF(self):
     def setHasCTF(self, value):
         self._hasCtf.set(value)
 
+    def hasFlexInfo(self):
+        return bool(self._flexProg.get())
```
Comment (pconesa) [inline pwem/objects/data.py:1322]: This will fail when self._flexProg is None

**Comment by pconesa** [comment]
please, confirm this is not needed anymore @DavidHerreros

**Comment by DavidHerreros** [comment]
Hi @pconesa 

Yes, these changes would not be needed anymore as now the data model of FlexHub is inside Scipion. We can close this PR.

**Pull request closed without merging**